src/checks/leading_character.rs

```diff
@@ -21,7 +23,10 @@ impl Check for LeadingCharacterChecker {
         {
             None
         } else {
-            Some(Warning::new(line.clone(), self.template.clone()))
+            Some(Warning::new(
+                line.clone(),
+                format!("{}: {}", self.name, self.template),
```

Comment:
Could you move this line to the `message` function as in the example?
https://github.com/mgrachev/dotenv-linter/blob/master/src/checks/unordered_keys.rs#L11